Add new python client library `ql` (#1765)
graphql/graphql.github.io: src/code/language-support/python/client/ql.md

---
name: ql
description: non intrusive python graphql client wrapped around pydantic.
url: https://dsal3389.github.io/ql/
github: https://github.com/dsal3389/ql
---

> Graphql client library, wrapped around pydantic classes for type validation,

Reviewer: ```suggestion
GraphQL client library, wrapped around pydantic classes for type validation,
```

> provide safe and simple way to query data from a graphql api.

Reviewer: ```suggestion
provide safe and simple way to query data from a GraphQL API.
```

> features:

Reviewer: ```suggestion
Features:

```

>   * python objects to valid graphql string

Reviewer: ```suggestion
  * python objects to valid GraphQL string
```

  * scalar query responses
  * typesafety

> ## install
> ```console

Reviewer: ````suggestion
## Install

```console
````

---
name: ql

> description: non intrusive python graphql client wrapped around pydantic.

Reviewer: ```suggestion
description: Non intrusive python GraphQL client wrapped around pydantic.
```

url: https://dsal3389.github.io/ql/
github: https://github.com/dsal3389/ql
---

GraphQL client library, wrapped around pydantic classes for type validation,

> provide safe and simple way to query data from a GraphQL API.

Reviewer: ```suggestion
provides a safe and simple way to query data from a GraphQL API.
```

> ---
> name: ql
> description: Non intrusive python GraphQL client wrapped around pydantic.
> url: https://dsal3389.github.io/ql/
> github: https://github.com/dsal3389/ql
> ---
> 
> GraphQL client library, wrapped around pydantic classes for type validation,
> provides a safe and simple way to query data from a GraphQL API.
> 
> Features:
> 
>   * python objects to valid GraphQL string
>   * scalar query responses
>   * typesafety
> 
> ## Install
> 
> ```console
> pip3 install pydantic-graphql
> ```

Reviewer: Formatted via Prettier Playground.

````suggestion
---
name: ql
description: Non intrusive python GraphQL client wrapped around pydantic.
url: https://dsal3389.github.io/ql/
github: https://github.com/dsal3389/ql
---

GraphQL client library, wrapped around pydantic classes for type validation,
provides a safe and simple way to query data from a GraphQL API.

Features:

- python objects to valid GraphQL string
- scalar query responses
- type-safety

## Install

```console
pip3 install pydantic-graphql
```
````
